src/buildOptions.ts

```diff
@@ -90,7 +90,8 @@ export const buildOptions = <T extends {}>(y: Argv<T>) => {
 				})
 				.option('outputDirectoryPath', {
 					type: 'string',
-					description: 'Output directory path for dry-run only',
+					description:
+						'(DEPRECATED, do not use) Output directory path for dry-run only',
```

Comment:
do we not replace this command with something new?

Reply:
no, it goes now into the homedir / cases / case hash digest (it was used anyway by the VSCE)